tests/test_util.py

```diff
@@ -16,3 +16,21 @@ def test_invalid_without_hash(self):
 
     def test_invalid_random_gene(self):
         assert not util.looks_like_rid('KRAS')
+
+
+class TestConvertAA3to1:
+    def test_substitution(self):
+        assert util.convert_aa_3to1('GP5:p.Leu113His') == 'GP5:p.L113H'
+
+    def test_non_matching_short_form(self):
+        assert util.convert_aa_3to1('GP5:p.Lys113His') == 'GP5:p.K113H'
+        assert util.convert_aa_3to1('CDK11A:p.Arg536Gln') == 'CDK11A:p.R536Q'
+
+    def test_truncation(self):
+        assert util.convert_aa_3to1('APC:p.Cys1405*') == 'APC:p.C1405*'
```

Comment:
Was this supposed to be:
```
util.convert_aa_3to1('APC:p.Cys1405Ter')
```
Showing Ter to * conversion?

Reply:
No, this was supposed to show that it would be ok with the `*` already bring there. Sometimes people who use 3-letter AA still use 1-letter for Ter